docs/hopsworksai/azure/aks_acr_integration.md

```diff
@@ -0,0 +1,229 @@
+    </a>
+    <figcaption>AKS network configuration</figcaption>
+  </figure>
+</p>
+
+Next go to integrations. Under container registry click Create new. 
+
+<p align="center">
+  <figure>
+    <a  href="../../../assets/images/hopsworksai/azure/aks-acr-create.png">
+      <img src="../../../assets/images/hopsworksai/azure/aks-acr-create.png" alt="AKS create ACR">
+    </a>
+    <figcaption>AKS create ACR</figcaption>
+  </figure>
+</p>
+
+Choose a name for the registry and check the SKU to be premium. Then press OK. 
+
+<p align="center">
+  <figure>
+    <a  href="../../../assets/images/hopsworksai/azure/aks-acr.png">
+      <img src="../../../assets/images/hopsworksai/azure/aks-acr.png" alt="AKS ACR configuration">
+    </a>
+    <figcaption>ACR configuration</figcaption>
+  </figure>
+</p>
+
+Next press review and create, then create the cluster.
+
+Because the kubernetes API service is private the hopsworks cluster must be able to reach it over a private network. There are two options to integrate with a private AKS cluster. The first option is to put the hopsworks cluster in a pre-defined virtual network with peering setup to the kubernetes network. The second option is to create a subnet inside the kubernetes virtual network where the hopsworks cluster will be placed.
+
+#### Step 1 option a Peering setup
+
+In order to create a peering first a virtual network for the hopsworks cluster must be setup. Go to virtual networks and press add.
+Choose a name for the new virtual network and select the same resource group you are planning to use for your hopsworks cluster.
+
+Next, go to the IP Addresses tab. Create an address space which does not overlap with the address space in the kubernetes network. In the previous example, the automatically created kubernetes network used the address space 10.0.0.0/8. Hence, the address space 172.18.0.0/16 can safely be used.
+
+Next click review and create, then create.
+
+Next go to the created virtual network and go to the peerings tab. Then click add. 
+
+<p align="center">
+  <figure>
+    <a  href="../../../assets/images/hopsworksai/azure/aks-peering.png">
+      <img src="../../../assets/images/hopsworksai/azure/aks-peering.png" alt="Virtual network peering">
+    </a>
+    <figcaption>Virtual network peering</figcaption>
+  </figure>
+</p>
+
+Choose a name for the peering link. Check the Traffic to remote virtual network as allow, and Traffic forwarded from remote virtual network to block.
+
+<p align="center">
+  <figure>
+    <a  href="../../../assets/images/hopsworksai/azure/aks-peering1.png">
+      <img src="../../../assets/images/hopsworksai/azure/aks-peering1.png" alt="Virtual network peering">
+    </a>
+    <figcaption>Virtual network peering configuration</figcaption>
+  </figure>
+</p>
+
+For the virtual network select the virtual network which was created by AKS, in our example this was hopsworksstagevnet154. Then press add.
```

Comment:
For the virtual network, select ... that was created by AKS. In our example, this was ...